index.js

```diff
@@ -80,8 +136,20 @@ genderCount(data);
 
 const promo20 = users => {
   // Your code goes here ...
+  
+  let balanceArr = users.filter(function(user) {
+    let balances = user.balance
+    let newBalance = balances.replace('$', '')
+    let newBalanceTwo = newBalance.replace(',', '')
+    return newBalanceTwo > 20000;
+  })
+  let [{firstName: name, balance: sum}, {firstName: secondName, balance: sumTwo}] = balanceArr;
+  console.log(`Dear ${name}, since your balance is ${sum}, you are eligible to apply for this awesome credit card.`)
+  console.log(`Dear ${secondName}, since your balance is ${sumTwo}, you are eligible to apply for this awesome credit card.`)
```

Comment:
These console.logs are throwing some errors due to variables not created. 
After you have the balanceArr with the filter. You can iterate over that array and do the console.log (or create a new array of strings) based on that information.